facade/src/main/scala/reactST/reactTable/TableMaker.scala

```diff
@@ -316,14 +436,38 @@ object TableMaker {
           val ascDesc = if (col.isSortedDesc.getOrElse(false)) "\u2191" else "\u2193"
           <.span(s" $index$ascDesc")
         } else TagMod.empty
-      <.th(props2Attrs(col.getHeaderProps()),
-           props2Attrs(col.getSortByToggleProps()),
-           cellClass,
-           col.render("Header"),
-           <.span(sortIndicator(col))
+
+      val headerProps = col.getHeaderProps()
+      val toggleProps = col.getSortByToggleProps()
+
+      // If both props have a "styles", only the last one gets used. Apparently, scalajs.react doesn't
+      // merge them. So, we have to combine them manually.
+      val styles = if (headerProps.hasOwnProperty("style") && toggleProps.hasOwnProperty("style")) {
```

Comment:
This is pretty ugly. Is there a better way?

Reply:
`Style` Monoid maybe?

Reply:
That's a good idea, but I'm not sure we could make it lawful in the case of duplicate keys. 

In any case, I should probably move this to a helper function of some sort.